ray-operator/test/sampleyaml/rayservice_test.go

```diff
@@ -0,0 +1,60 @@
+package sampleyaml
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	. "github.com/ray-project/kuberay/ray-operator/test/support"
+)
+
+func TestRayService(t *testing.T) {
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "ray-service.sample.yaml",
+		},
```

Comment:
There are many other RayService yaml files under `ray-operator/config/samples/` and its subfolders. Could you also check those files that:

- Do not use too much resources
- Do not rely on external services
- No autoscaling

Reply:
I found that tests sometimes fail because the worker's Readiness probe failed with a `success` message, causing the Ray Cluster to be unable to reach a Ready state, which leads to test failures. Is there any other way to resolve this issue?

![image](https://github.com/user-attachments/assets/48fc4c48-3d79-4152-b9ee-d271e2ac7021)